capacity-dashboard/README.md

```diff
@@ -0,0 +1,82 @@
+
+### Interpreting the current usage panel
+
+Current usage is a highly volatile metric that varies by the second based on volumes of requests on the running applications. It is not suitable for making load predictions and deciding on whether the cluster should scale. Nonetheless this metric is included in the capacity management dashboard for two reasons:
+
+1. having this metrics allows knowing at glance if we are running out of capacity for the actual used resources. Notice that we cannot use this metrics for monitoring the capacity of the individual nodes. In fact this metric averages the nodes and even if the metric looks healthy, we may have a node that is running out of capacity. Other means must be used to monitor the health of the nodes.
+2. if this metric diverges significantly from what measured by the sum of requests, it means that our developers are badly estimating their work load. To fix the issue one should dig deeper (maybe with additional dashboard), discover which pods are responsible for the divergence and have the developers that own those nodes fix the request for those nodes.
+
+### Interpreting the request vs allocatable ratio panel
+
+This ratio shows what is being requested (estimates of what we need) vs what the node group has. Assuming we have good estimates, this is a much better signal to scale upon because it's less volatile (it changed only when pods are added/removed). With some approximation, this is also the signal autoscaler, assuming you have configured you machinesets to correspond with nodegroups. But contrary to autoscaler, which is very reactive adn will add a new node only when a new pod would make the ration exceed 100%, we can be more proactive using this panel and decide for example to scale when 80% of the resources have are being requested. This metrics for scaling the cluster works well when we can provision a new node relatively quickly, so by and large when nodes are VMs.
+
+### Interpreting the quota vs allocatable ratio panel
+
+This ration shows what we have promised to the team we can give them in terms of resources versus what we have. This is a slow moving ratio because it changes only when new quotas are granted, which should more or less coincide with new teams being onboarded to the cluster. This metric can be used to make long term (and therefore non-precise) forecast of what is going to be needed. This metrics is best suited when our node provisioning lifecycle is very long: weeks or months. This likely happens when the nodes are physical machines.
+With this ratio is possible and ok to go above 100%. In that situation the cluster is over committed. Over committing the cluster is generally fine if we can safely assume that not all the projects will use all of their quotas at the same time. Each organization will have to find its own sweet spot in terms of over committing the cluster.
+
+## Installation
+
+Install the openshift-state-metrics additional metrics here as follows
```

Comment:
@raffaelespazzoli FYI, openshift state metrics is already installed by the custom dashboard project.

Reply:
are we using the latest with that approach? I saw that Shah was not using the latest.

Reply:
the latest push should address the concern.
If there was a way for applier to get the manifests directly from the openshift state metrics repo, I would recommend doing that.